runner/steps.ts

```diff
@@ -157,6 +158,16 @@ function runBrowsers(context: Context) {
             errors.push(error);
           }));
       runners.push(runner);
+      if (browserDef.browserName === 'safari') {
+        // Control to Safari must be serialized. We can't launch two instances
+        // simultaneously, because security lol.
+        // https://webkit.org/blog/6900/webdriver-support-in-safari-10/
+        waitFor = runner.donePromise.catch(() => {
+          // The next runner doesn't care about errors, just wants to know when
+          // it can start.
+          return null;
```

Comment:
I initially had a return expression rather than a body here, but then it looked awkward to add the comment about why I was doing that. Then I remove the return and clang-format formats it kinda ugly. It's a little dumb, but I don't think it's wrong or unclear. ¯\\_(ツ)_/¯